source/common/http/filter/gzip_filter.cc

```diff
@@ -2,9 +2,18 @@
 
 #include <regex>
 
+#include "common/common/macros.h"
+
 namespace Envoy {
 namespace Http {
 
+namespace {
```

Comment:
What's the purpose of this namespace?

Reply:
I just realized that this doesn't apply to c++11 or up anymore. I will remove it.